sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -474,8 +474,6 @@ def test_partial_dependence_X_list(estimator):
     partial_dependence(estimator, list(X), [0])
 
 
-# TODO: Remove in 0.24 when DummyClassifier's `strategy` default updates
```

Comment:
these 2 aren't related to module deprecation but I think they're the only ones remaining so I'll squeeze them here instead of opening another PR